src/content/api/loaders.md

```diff
@@ -471,3 +509,96 @@ Hacky access to the Compiler object of webpack.
 ### `this._module`
 
 Hacky access to the Module object being loaded.
+
+
+## Error Reporting
+
+There are some ways to throw out errors from inside a loader:
+
+### Using `this.emitError`
+
+It just report errors but not interrupt module compiling.
+
+see [this.emitError](/api/loaders/#this-emiterror).
+
+### Using `throw` (or other uncaught exception)
+
+Throw out an uncaught error while loader are running will cause current module compiling failed.
```

Comment:
`...while a loader is running` is correct.
Correct word is `fail`.

Reply:
or: Throwing an error while loader is running will cause current module compilation failure.